<?xml version="1.0"?>
<launch>

    <param name="maxlen"          type="double" value="10.0" />    
    <param name="minlen"          type="double" value="0.0" />    
    <param name="max_ang_FOV"     type="double" value="2.7" />
    <param name="min_ang_FOV"     type="double" value="0.5" />
    <param name="pcTopic"         type="str"    value="/ouster/points" />
    <param name="imgTopic"        type="str"    value="/camera/color/image_raw" />
    
    <param name="x_resolution"    type="double" value="0.15" /> <!-- X resolution of lidar on image plane-->
    <param name="y_interpolation" type="double" value="10.0" /> <!-- Y interpolation of lidar layers on image plane-->
    <param name="ang_Y_resolution"    type="double" value="2.05" />

    <group ns="matrix_file">
        <rosparam file="$(find lidar_camera_fusion)/cfg/cfg_params.yaml" />
    </group>
    
    <node pkg="lidar_camera_fusion" type="lidar_camera_node" name="lidar_camera_node" output="screen"/>

<!-- osuter os1-u 128  -->
  <include file="$(find ouster_ros)/launch/sensor.launch" />
<!-- camera -->
<include file="$(find realsense2_camera)/launch/rs_camera.launch" />

<!-- rviz
<node type="rviz" name="rviz" pkg="rviz" args="-d $(find lidar_camera_fusion)/rviz/lidar_camera.rviz" />-->
  
</launch>
